ksql-udf-quickstart/pom.xml

```diff
@@ -2,17 +2,16 @@
 <!--
   ~ Copyright 2018 Confluent Inc.
```

Comment:
nit: looks like this file and the four below it (archetype-metadata.xml, ksql-udf-quickstart pom, ReverseUdf, and SummaryStatsUdaf) were added in 2019 despite having copyrights that say 2018. Feel free to ignore though since I can't imagine this making a difference later.